src/main/webapp/assets/modules/chart/chart.con.js

```diff
@@ -5,6 +5,17 @@ angular.module('Chart').controller('chartController', function ($scope, jsonValu
   var chartFactory = chartService.getChartFactory();
 
   $scope.$on(events.terms, function (event, terms) {
+    $.each(terms, function(i, term) {
+      if ($.isNumeric(term.averageSalaryMin) &&  $.isNumeric(term.averageSalaryMax)) {
+        term.salRange = sprintf("%(averageSalaryMin)s - %(averageSalaryMax)s", term);
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'sprintf' is not defined.